From d16ff5c412b90b2766be97a5d3d689c5a18528d3 Mon Sep 17 00:00:00 2001
From: usa <usa@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
Date: Fri, 29 Apr 2016 11:12:17 +0000
Subject: [PATCH] merge revision(s) 54706:

	* test/ruby/test_time_tz.rb: Tests depends on Europe/Moscow removed
	  to avoid test failures due to the tzdata change.
	  https://github.com/eggert/tz/commit/8ee11a301cf173afb0c76e0315b9f9ec8ebb9d95
	  Found by naruse.


git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/branches/ruby_2_2@54815 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
---
 ChangeLog                 |  7 +++++++
 test/ruby/test_time_tz.rb | 13 -------------
 2 files changed, 7 insertions(+), 13 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index fe09b1e..1351c74 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,10 @@
+Fri Apr 29 20:11:38 2016  Tanaka Akira  <akr@fsij.org>
+
+	* test/ruby/test_time_tz.rb: Tests depends on Europe/Moscow removed
+	  to avoid test failures due to the tzdata change.
+	  https://github.com/eggert/tz/commit/8ee11a301cf173afb0c76e0315b9f9ec8ebb9d95
+	  Found by naruse.
+
 Tue Apr 26 13:38:31 2016  NAKAMURA Usaku  <usa@ruby-lang.org>
 
 	* README.EXT{,.ja}: `nul` should be uppercase.
diff --git a/test/ruby/test_time_tz.rb b/test/ruby/test_time_tz.rb
index fb1ab9b..96a03b7 100644
--- a/test/ruby/test_time_tz.rb
+++ b/test/ruby/test_time_tz.rb
@@ -169,13 +169,6 @@ def test_europe_lisbon
     }
   end if has_lisbon_tz
 
-  def test_europe_moscow
-    with_tz(tz="Europe/Moscow") {
-      assert_time_constructor(tz, "1992-03-29 00:00:00 +0400", :local, [1992,3,28,23,0,0])
-      assert_time_constructor(tz, "1992-03-29 00:59:59 +0400", :local, [1992,3,28,23,59,59])
-    }
-  end
-
   def test_pacific_kiritimati
     with_tz(tz="Pacific/Kiritimati") {
       assert_time_constructor(tz, "1994-12-31 23:59:59 -1000", :local, [1994,12,31,23,59,59])
@@ -355,12 +348,6 @@ def self.gen_zdump_test(data)
 Europe/London  Sun Aug 10 01:00:00 1947 UTC = Sun Aug 10 02:00:00 1947 BST isdst=1 gmtoff=3600
 Europe/London  Sun Nov  2 01:59:59 1947 UTC = Sun Nov  2 02:59:59 1947 BST isdst=1 gmtoff=3600
 Europe/London  Sun Nov  2 02:00:00 1947 UTC = Sun Nov  2 02:00:00 1947 GMT isdst=0 gmtoff=0
-Europe/Moscow  Sat Jan 18 23:59:59 1992 UTC = Sun Jan 19 01:59:59 1992 MSK isdst=0 gmtoff=7200
-Europe/Moscow  Sun Jan 19 00:00:00 1992 UTC = Sun Jan 19 03:00:00 1992 MSK isdst=0 gmtoff=10800
-Europe/Moscow  Sat Mar 28 19:59:59 1992 UTC = Sat Mar 28 22:59:59 1992 MSK isdst=0 gmtoff=10800
-Europe/Moscow  Sat Mar 28 20:00:00 1992 UTC = Sun Mar 29 00:00:00 1992 MSD isdst=1 gmtoff=14400
-Europe/Moscow  Sat Sep 26 18:59:59 1992 UTC = Sat Sep 26 22:59:59 1992 MSD isdst=1 gmtoff=14400
-Europe/Moscow  Sat Sep 26 19:00:00 1992 UTC = Sat Sep 26 22:00:00 1992 MSK isdst=0 gmtoff=10800
 Pacific/Kiritimati  Sun Jan  1 09:59:59 1995 UTC = Sat Dec 31 23:59:59 1994 LINT isdst=0 gmtoff=-36000
 Pacific/Kiritimati  Sun Jan  1 10:00:00 1995 UTC = Mon Jan  2 00:00:00 1995 LINT isdst=0 gmtoff=50400
 End
